<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtHelpConfigEditDialog</class>
 <widget class="QDialog" name="QtHelpConfigEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>125</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string comment="@label:chooser">Icon:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KIconButton" name="qchIcon">
     <property name="toolTip">
      <string comment="@info:tooltip">Select an icon</string>
     </property>
     <property name="iconSize">
      <number>16</number>
     </property>
     <property name="strictIconSize">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string comment="@label:textbox">Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="qchName">
     <property name="toolTip">
      <string comment="@info:tooltip">Enter a name</string>
     </property>
     <property name="placeholderText">
      <string comment="@info:placeholder">Select a name...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string comment="@label:textbox">Path:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KUrlRequester" name="qchRequester">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string comment="@info:tooltip">Select a Qt Help file...</string>
     </property>
     <property name="filter">
      <string notr="true">*.qch|Qt Compressed Help (.qch)</string>
     </property>
     <property name="placeholderText">
      <string comment="@info:placeholder">Select a Qt Help file...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>KIconButton</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>KUrlRequester</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QtHelpConfigEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QtHelpConfigEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
